content/en/data_security/data_retention_periods.md

```diff
@@ -124,8 +124,8 @@ attributes:
        - **Netflow**: 30 days
        - **SNMP traps**: 15 days
   - product: Cloud Network Monitoring
-    data_type: |
-       - **Netflow**: 14 days
+    data_type: | 
+          - **Network traffic**: 14 days
```

Comment:
```suggestion
    data_type: |
        - **Network traffic**: 14 days
```
I don't know if the whitespace matters, but let's make it consistent to be safe.